views/admin.ejs

```diff
@@ -4,17 +4,8 @@
     <meta name="viewport" content="initial-scale=1, maximum-scale=1">
     <title>Talk - Coral Admin</title>
     <style media="screen">
-      body, #root {
-        width: 100%;
-        height: 100%;
-        margin: 0;
-        background-color: #FAFAFA;
-        font-family: 'Roboto', sans-serif;
-      }
-
-      #root > div {
-        height: 100%;
-      }
+      html, body, #root, #root > div { min-height: 100%; }
```

Comment:
Scrolling issues on several pages with these changes. It took me a lot of trial and error to find the correct css controlling `height`. Try to find a solution that is very targeted, solves your issue but leaves the rest in tact.

Reply:
The reason for the difficulties was to properly support `react-virtualized`.

Reply:
For instance you can just increase the height of the paragraph on the particular views that are causing problems.

Reply:
Better yet, Just give `Layout` just the right amount of min-height to cover the popup, but not more.

Reply:
Could you be more especific on how `min-height` could introduce errors? I see the queues perfectly working. Maybe I'm missing something. (the `height` is still `inherit` as you marked it on the first place)